.github/workflows/docs-toc.yml

```diff
@@ -1,20 +0,0 @@
-# Verify that important design docs have ToC entries.
-name: Check documentation ToC
-on:
-  pull_request:
-    push:
-      branches:
-        - main
-
-jobs:
-  check:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: technote-space/get-diff-action@v6
-        with:
-          PATTERNS: |
-            docs/architecture/**
-            docs/rfc/**
-      - run: make check-docs-toc
```

Comment:
This workflow would still be useful actually as it purely ensures that, if someone's submitting an RFC or ADR, that it has a corresponding link in either the RFC or ADR table of contents.

If we remove this here I'd recommend that, in a follow-up PR where we restructure the ADRs/RFCs, we reintroduce it. Otherwise rather just keep this workflow, the `check-docs-toc` Makefile target, and `docs/presubmit.sh` script.

Reply:
OK, I've reverted these changes